rules/sinks/third_parties/sdk/google/javascript.yaml

```diff
@@ -401,5 +401,5 @@ sinks:
       - "analytics.google.com"
```

Comment:
Domain can be changed to `tagmanager.google.com`.

Reply:
yes will change this.